localisation/touhou_events_l_english.yml

```
@@ -2156,6 +2156,11 @@
  flavor_clp.15.a: "Let them come! We will decimate them!"
  flavor_clp.15.a.tt: "Every owned province with the §YLunarian§! culture will get '§YLunarians' Last Stand§!' for §Y30§! years, giving the following effect:\nYears of Separatism: §G-30§!"
  flavor_clp.15.b: "We should negotiate with them before they pick up their weapons."
+ flavor_clp.16.t: "The Clown's advance"
```

Title capitalization
```suggestion
 flavor_clp.16.t: "The Clown's Advance"
```